readme.md

```diff
@@ -59,7 +59,9 @@ Returns an array of `inputs` filtered based on the `patterns`.
 
 ### matcher.isMatch(input, pattern, options?)
 
-Returns a `boolean` of whether the `input` matches the `pattern`.
+Accepts either a string or array of strings for both `input` and `pattern`.
+
+Returns a `boolean` of whether any given `input` matches every given `pattern`.
```

Comment:
This change needs to be reflected in in index.d.ts too.

Reply:
Done, updated the `param`/`returns` docs.